docker/terraform/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG TERRAFORM_VERSION
 WORKDIR /build/
 RUN GOBIN=$(pwd) go install github.com/hashicorp/terraform@v${TERRAFORM_VERSION}
 
-FROM gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine@sha256:d5da0344b23d03a6f2728657732c7a60300a91acaad9b8076c6fd30b1dfe1ff4
```

Comment:
@michaelkedar I standardized on the same image everywhere, can you confirm that's okay to do here?

Reply:
I'm not actually sure what's specifically required from this image.
The [original source of this Dockerfile](https://github.com/GoogleCloudPlatform/cloud-builders-community/blob/master/terraform/Dockerfile) used `gcr.io/cloud-builders/gcloud`, so if that had some special sauce needed for running in Cloud Build, it wasn't necessary.

So I'd say it's *probably* fine.

Reply:
Also note that this image isn't automatically built/pushed.